src/tests/architect_tests/test_integration.py

```diff
@@ -367,22 +374,40 @@ def test_integration_simple():
         expected_matrix_multiplier=1,
     )
 
+    print(matrices_records)
+    assert len(matrices_records) == 4
+    assert matrices_records[0][0] == '8af84bde5a09e9adac63a384d924aab3' #The hash uuid
```

Comment:
I do this sometimes but am trying to cut down on it; asserting a specific value of the hash is too detailed for this test, and kind of brittle. We do test the hashing algorithm elsewhere, and doing it in a test this high-level can be prone to breakage.

I think a version of testing the hash in this test that would be appropriate for the scope of the test would be testing it against the matrix on disk. But that would be done in the `basic_integration_test` functions, and I've already added a comment like this above.